### PR TITLE
chore: update v10-info

### PR DESCRIPTION
**Pull request by tujoworker**
Preview of the docs: https://eufemia-a5hj9ltie-eufemia.vercel.app/uilib/about-the-lib/releases/eufemia/v10-info/

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases.mdx [hunk 1]
```diff
@@ -15,7 +15,7 @@ You may also have a look at the [<Icon icon={GithubLogo} size="default" /> **Git
 
 Here you find an overview of all major releases (versions) and changes, including migration guides:
 
-- [**@dnb/eufemia@v10**](/uilib/about-the-lib/releases/eufemia/v10-info) _February, 24. 2023_
+- [**@dnb/eufemia@v10**](/uilib/about-the-lib/releases/eufemia/v10-info) _Mars, 10. 2023_
```
[the remark this answers is not in the document]
Comment (tujoworker) [reply packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases.mdx]: This has been resolved by a rebase form the v10 branch. Also, restructured the whole doc. Hope you can have another read through. 🙏

**Comment by tujoworker** [comment]
@langz I restructured the whole doc. Hope you can have another read through. 🙏

File: packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (langz) [reply packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases.mdx]: I'll take a look later today👀

File: packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases/eufemia/v10-info.mdx [hunk 2]
```diff
@@ -1,41 +1,81 @@
...
-    - [Deprecations](#deprecations)
-  - [TypeScript](#typescript)
-  - [Install](#install)
 
-## Migration
+</Accordion>
+
+## How to migrate
+
+v10 of [@dnb/eufemia](https://www.npmjs.com/package/@dnb/eufemia) contains _breaking changes_. As a migration process from v9, you should follow all of the above guides.
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases/eufemia/v10-info.mdx:41]:
```suggestion
v10 of [@dnb/eufemia](https://www.npmjs.com/package/@dnb/eufemia) contains _breaking changes_. As a migration process from v9, you should follow all of the below guides.
```

Not sure if `above` or `below` is correct. The TOC is of course above, but the "content" is sort of below 😂

File: packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases/eufemia/v10-info.mdx [hunk 3]
```diff
@@ -1,41 +1,81 @@
...
+### TypeScript
+
+1. Updated multiple types from `string | boolean` to `boolean`, as there was a lot of properties who should only support boolean values and not strings.
+   1. Examples of changes to do would be to find Eufemia components using `"false"` or `"true"`, and replace it with boolean values. i.e., `vertical="false"` to `vertical={false}` or `selectall="true"` to `selectall={true}`.
+2. `import { LocaleProps, DataAttributeTypes, DynamicElement } from '@dnb/eufemia/shared/interfaces'` was removed, and moved to `@dnb/eufemia/shared/types`.
+
+## Discontinued Internet Explorer (IE) support
 
-## Features
+The support for Internet Explorer (IE) was removed, as Microsoft formally ended support for IE in June, 2022. Have a look at the updated table about supported [browsers and paltforms](/uilib/usage/#supported-browsers-and-platforms).
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases/eufemia/v10-info.mdx:68]:
```suggestion
The support for Internet Explorer (IE) was removed, as Microsoft formally ended support for IE in June, 2022. Have a look at the updated table about supported [browsers and platforms](/uilib/usage/#supported-browsers-and-platforms).
```

Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases/eufemia/v10-info.mdx:68]:
```suggestion
The support for Internet Explorer (IE) was removed, as Microsoft formally ended support for IE in June, 2022. Have a look at the supported [browsers and platforms](/uilib/usage/#supported-browsers-and-platforms).
```
As I believe we removed the actual "table".

File: packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases/eufemia/v10-info.mdx [hunk 4]
```diff
@@ -1,41 +1,81 @@
...
 
-1. [Browser support](/uilib/usage/#supported-browsers-and-platforms) has been updated to support modern browsers only (es6).
+With that change, Eufemia will support modern browsers that supports ES6.
 
-## Web Components
+1. The helpers `isIE11` and `IS_IE11` where removed.
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases/eufemia/v10-info.mdx:72]:
```suggestion
1. The helpers `isIE11` and `IS_IE11` were removed.
```
```suggestion
1. The helpers `isIE11` and `IS_IE11` was removed.
```

Not 100% sure of were vs. was.

File: packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases/eufemia/v10-info.mdx [hunk 5]
```diff
@@ -61,76 +101,109 @@ $layout-large: map-get($breakpoints, 'large');
...
+
+1. If you did import them as so:
+
+```js
+import '@dnb/eufemia/style'
+```
+
+then you don't need to make any changes.
+
+2. If you did import `/components` etc. – then you have to change it to the above:
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases/eufemia/v10-info.mdx:114]:
```suggestion
2. If you did import `/components` etc. – then you have to change it to the the import above(see 1.):
```

File: packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases/eufemia/v10-info.mdx [hunk 6]
```diff
@@ -61,76 +101,109 @@ $layout-large: map-get($breakpoints, 'large');
...
 
 ```js
 import '@dnb/eufemia/style/core'
 import '@dnb/eufemia/style/themes/ui'
 ```
 
-Or as CSS imports:
+3. If you did import the styles as CSS files, then you have to change it to the above:
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases/eufemia/v10-info.mdx:121]:
```suggestion
3. If you did import the styles as CSS files, then you have to change it to the the import above(see 1.):
```

File: packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases/eufemia/v10-info.mdx [hunk 7]
```diff
@@ -61,76 +101,109 @@ $layout-large: map-get($breakpoints, 'large');
...
 import '@dnb/eufemia/style/themes/theme-ui/dnb-theme-components.min.css'
 import '@dnb/eufemia/style/themes/theme-ui/dnb-theme-basis.min.css'
 ```
 
-In detail:
+More details about the change:
 
 - The package `dnb-theme-ui` was renamed to `dnb-theme-basis`.
 - The package `dnb-ui-components` was renamed and moved inside a theme `/style/themes/theme-ui/dnb-theme-components.*`.
 - `dnb-ui-tags` was renamed and moved form `/style/dnb-ui-tags.*` to `/style/themes/theme-ui/dnb-theme-tags.*`.
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases/eufemia/v10-info.mdx:133]:
```suggestion
- `dnb-ui-tags` was renamed and moved from `/style/dnb-ui-tags.*` to `/style/themes/theme-ui/dnb-theme-tags.*`.
```

**Review by langz — APPROVED** [review]
Looks great, I just added a few minor comments.

I did also observe that "links" to the "actual changed components" like Table, in the TOC, didn't work as expected. It did not send me to the docs of that actual component.
"Pressing Table in the TOC don't scroll me to the docs".

Here's a video of what I'm trying to explain/show(it works for "[Pagination](https://eufemia-a5hj9ltie-eufemia.vercel.app/uilib/components/pagination) and [InfinityScroller](https://eufemia-a5hj9ltie-eufemia.vercel.app/uilib/components/pagination/infinity-scroller)", but not the other ones I try to click):


https://user-images.githubusercontent.com/1359205/232016440-60e787e8-7a31-426b-8f45-0001a39ec6e6.mov

File: packages/dnb-design-system-portal/src/docs/contribute/style-guides/naming.mdx [hunk 8]
```diff
@@ -42,7 +37,7 @@ The DNB Design System Eufemia uses the following naming conventions.
 - Event names use **camel case**.
 - They have to describe what they are aiming to do. Like: `onClick`
 
-**NB:** Existing components use **snake case** (`on_click`) – but you are free to use **camel case**.
+**NB:** Existing React components use **snake case** (`on_click`) – but will be converted to use **camel case** in future releases. The reason was to support Web Components – but the support has been discontinued in [v10](/uilib/about-the-lib/releases/eufemia/v10-info/).
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/contribute/style-guides/naming.mdx:40]:
```suggestion
**NB:** Existing React components use **snake case** (`on_click`) – but will be converted to use **camel case** in future releases. The reason for using **snake case** was to support Web Components – but the support has been discontinued in [v10](/uilib/about-the-lib/releases/eufemia/v10-info/).
```

**Comment by tujoworker** [comment]
> "Pressing Table in the TOC don't scroll me to the docs".

@langz Nice finding! I have added an [commit](https://github.com/dnbexperience/eufemia/pull/2037/commits/fe298fb1b94839fca3fac5e9497f051a8266fb4f) that should solve the issue.

**Pull request merged**